irods_http_client/user_group_operations.py

```diff
@@ -4,7 +4,7 @@
 class UsersGroups:
     def __init__(self, url_base: str):
         """
-        Initializes DataObjects with a base url. 
+        Initializes UsersGroups with a base url. 
```

Comment:
This change should be part of the other commit.

Reply:
Moved this change to the appropriate  commit